model-optimizer/mo/middle/passes/mean_scale_values.py

```diff
@@ -20,62 +20,130 @@
 from mo.middle.pattern_match import apply_pattern
 
 
-def move_scaleshift_to_preprocess_action(graph, match):
+def move_mean_values_to_preprocess_action(graph, match):
     mean_values = {}
+    if 'const_mul_output' in match and match['const_mul_output'].has('value'):
+        weights = match['const_mul_output'].value
+        if any([x != 1 for x in weights]):
+            return
+
```

Comment:
Here we already know that the Mul operation has one of the inputs equal to 1, so we can remove `mul` node and re-connect it's input and output properly. No need to do this in line 41, 48.